### DISCUSSION TITLE
Hide/change commit name

### DISCUSSION THREAD

**Discussion by rschirin** [community/community#23817] [New to GitHub]
<p>Hi, I’m contributing to a repository and I’m using Visual Studio Code to commit my changes.<br>
I saw that commits are linked to my name and last name. Is there a way to avoid it? For instance I can accept to see my username. Furthermore I’m not sure if this behavior is due to git on my pc, to my github account or visual studio code.<br>
Anyone can address me?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/rschirin">@rschirin</a>, Welcome to the community! You may need to configure your Git username and email for VSCode:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <a href="https://azuredevopslabs.com/labs/azuredevops/github/#task-1-configuring-visual-studio-code" target="_blank">azuredevopslabs.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://azuredevopslabs.com/labs/azuredevops/github/#task-1-configuring-visual-studio-code" target="_blank">Version Controlling with Git in Visual Studio Code and GitHub</a></h3>

<p>Check out the tutorial 'Version Controlling with Git in Visual Studio Code and GitHub' and many more at https://aka.ms/devopslabs</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I hope this helps!</p>
